examples/ZemiSmart/Makefile

```diff
@@ -0,0 +1,18 @@
+PROGRAM = light
+
+EXTRA_COMPONENTS = \
+	extras/http-parser \
+	$(abspath ../../components/wolfssl) \
+	$(abspath ../../components/cJSON) \
+	$(abspath ../../components/homekit)
+
+LIBS ?= hal m
```

Comment:
Please change it to `LIBS += m` and move it after `include $(SDK_PATH)/common.mk`. The hal library is something provided by framework and it would be nice to have incapsulated there in case it changes.